### DISCUSSION TITLE
No idea where to report a bug ?

### DISCUSSION THREAD

**Discussion by DaveTheBuskerUK** [supermerill/SuperSlicer#4272] [Community contribution]
I run SuperSlicer on Linux Mint using the AppImage, up until SuperSlicer-ubuntu_18.04-gtk2-2.5.59.8 I never had this error but it shows up with .9 and the new .10 releases, same file, same other open aps etc, the only change is the version of SS, if I open the .8 version with the same file it slices fine.

"
SuperSlicer has encountered an error. It was likely caused by running out of memory. If you are sure you have enough RAM on your system, this may also be a bug and we would be glad if you reported it.
std::bad_alloc
"

For now I'm just going to revert back to version .8 and try each new release.

Sorry for posting this here but I don't have a clue how to properly report a bug, I am happy to move / delete this or provide any additional info needed.

Keep up the good work, SS has been my go to slicer for a few years now.

Regards,

Dave.

**Comment by darco415** [comment]
Well you are on github so your're very close..the tab above, "issues" is what your're looking for. Also there are  users who  reported having memory issues  (like me) but I havent seen it for the latest edition (.10). I have Linux Mint and it slices fine
